docs/connect-storefront-to-platform-v3.md

```diff
@@ -0,0 +1,40 @@
+# Connect Storefront to Platform
+
+* Deploy the latest storefront version as described [there](https://virtocommerce.com/docs/vc2devguide/deployment/storefront-deployment)
+
+Note if you deploy the Storefront on local environment with the Platform you should host it with IIS
+
+* Make changes  in  `appsettings.json`
+
+```json
+...
+ "Endpoint": {
+     //Specify platform url
+     "Url": "https://localhost:5001",
+     //Comment the follow settings
+     //"AppId": "...",
+     //"SecretKey":"...",
+
+     //Uncomment the follow settings
+      "UserName": "admin",
+      "Password": "store",
```

Comment:
switch the "Comment" with "Uncomment"??

Reply:
User name and password used to connect to Platform V3

"UserName": "admin",
"Password": "store",
lines commented  by default in Storefront appsettings.json